JSDemos/Demos/Lookup/EventHandling/description.md

```diff
@@ -1 +1,9 @@
-This demo illustrates how to&nbsp;perform a&nbsp;custom action when the lookup value is&nbsp;changed. Use the **onValueChanged** event handler to&nbsp;process the new lookup value.
\ No newline at end of file
+To handle the Lookup's value change, specify the [onValueChanged](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#onValueChanged) handler. This demo uses it to display the picture and text for the selected item.
+
+If you want to change event handler at runtime or attach several handlers to the 
+[valueChanged](/Documentation/ApiReference/UI_Components/dxLookup/Events/#valueChanged) event, use the [on(eventName, eventHandler)](/Documentation/ApiReference/UI_Components/dxLookup/Methods/#oneventName_eventHandler) method to subscribe to this event. This approach is more typical of jQuery. For Angular and Vue, you can use two-way data bindings. To do this in React, create a state variable.
+
+The Lookup applies the selection instantly. If you want users to click the Apply button instead, set the [applyValueMode](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#applyValueMode) property to *"useButtons"*.
```

Comment:
We haven't mentioned selection yet, so this paragraph looks out of place. Why do we even mention this property? It's not in the demo, and it is related to the topic only barely.